chain/jsonrpc/res/epoch_info.html

```diff
@@ -31,27 +32,75 @@
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script>
+        function convertTime(millis) {
+            var seconds = Math.floor(millis / 1000);
```

Comment:
We probably should prefer `let` and `const` rather than `var`. They have lexical scoping which is easier to understand. Furthermore, `const` prevents rebinding of the variable.

Reply:
I think so too: `let` and `const` are better options as long as they are available.

Reply:
done.